tests/build_/test_arrangements.py

```diff
@@ -1060,11 +1072,14 @@ def test_pulp_publish(self, osbs_with_pulp):  # noqa:F811
 
         assert args == expected_args
 
-    def test_pulp_pull(self, osbs_with_pulp):  # noqa:F811
+    @pytest.mark.parametrize('osbs', [  # noqa:F811
+        {'platform_descriptors': None, 'additional_config': get_pulp_additional_config(),
```

Comment:
I see this definition many times, could it be constant at module level?

Reply:
Done